### PR TITLE
fix(flowbuilder): fix wait on issue with new rtl procedure

### PR DESCRIPTION
**Pull request by billsomen**
This PR is about fixing known issues after merging [this PR](https://github.com/botpress/studio/pull/61)
The issue : The *WaitOn user input* is always displayed.
The solution: 
- I reverted the RTL updates 
- I also removed unnecessary props from the store
- I fetched the contentLang from the store on the Node component (cause of the issue) directly from the store object. 

(below how it looks like now)
image

### CODE REVIEW

File: packages/studio-ui/src/web/components/SmartInput/index.tsx [hunk 1]
```diff
@@ -140,10 +141,12 @@ class SmartInput extends React.Component<ConnectedProps, State> {
     if (this.props.singleLine) {
       plugins.push(createSingleLinePlugin())
     }
+
+    isRTLLocale(this.props.contentLang)
```
Comment (allardy) [inline packages/studio-ui/src/web/components/SmartInput/index.tsx:145]: is this necessary ?

File: packages/studio-ui/src/web/views/FlowBuilder/common/action.tsx [hunk 2]
```diff
@@ -6,11 +6,12 @@ import Markdown from 'react-markdown'
 import { connect } from 'react-redux'
 import { fetchContentItem, refreshFlowsLinks } from '~/actions'
 
-import { isMissingCurlyBraceClosure } from '../../../components/Util/form.util'
+import { isMissingCurlyBraceClosure } from '~/components/Util/form.util'
 import withLanguage from '../../../components/Util/withLanguage'
 import { ActionPopover } from './actionPopover'
 
 import style from './style.scss'
+import { isRTLLocale } from '~/translations'
```
Comment (allardy) [inline packages/studio-ui/src/web/views/FlowBuilder/common/action.tsx:15]: please fix the lint warning

File: packages/studio-ui/src/web/views/FlowBuilder/common/action.tsx [hunk 3]
```diff
@@ -128,7 +127,14 @@ class ActionItem extends Component<Props> {
     }
 
     return (
-      <div className={classnames(this.props.className, style['action-item'], style.msg, isRTLContentLang ? style.rtl : null)}>
+      <div
+        className={classnames(
+          this.props.className,
+          style['action-item'],
+          style.msg,
+          isRTLLocale(this.props.contentLang) ? style.rtl : null
```
Comment (allardy) [inline packages/studio-ui/src/web/views/FlowBuilder/common/action.tsx:135]: with classnames: ``[style.rtl]: isRTLLocale``

**Review by J-FMartin — DISMISSED** [review]
no need to ask me for PRs going forward, this should remain in the Eng team

**Review by allardy — APPROVED** [review]
LGTM

**Pull request merged**